shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-postgresql/src/main/java/org/apache/shardingsphere/data/pipeline/postgresql/ingest/wal/decode/TestDecodingPlugin.java

```diff
@@ -153,36 +155,46 @@ private String readColumnType(final ByteBuffer data) {
     }
     
     private Object readColumnData(final ByteBuffer data, final String columnType) {
-        if (columnType.startsWith("numeric")) {
-            return new BigDecimal(readNextSegment(data));
-        }
         if (columnType.startsWith("bit") || columnType.startsWith("bit varying")) {
             return readNextSegment(data);
         }
```

Comment:
Does "bit" and "bit varying" need to do the same change?

Reply:
Yes, I will fix it.